File: SocialAppFlutter/social_app/lib/services/auth/google_sign_in_request.dart
class GoogleSignInRequest {
  final String email;
  final String firstname;
  final String lastname;
  final List<int>? profileImageBytes;

  GoogleSignInRequest(this.email, this.firstname, this.lastname, this.profileImageBytes);

  Map<String, dynamic> toJson() =>
      {'email': email, 'firstname': firstname, 'lastname': lastname, 'profileImageBytes': profileImageBytes
      };
}

File: APIs/AuthService/AuthService/DTOs/GoogleSignInRequest.cs
﻿namespace AuthService.DTOs
{
    public class GoogleSignInRequest
    {
        public string Email { get; set; } = string.Empty;
        public string Firstname { get; set; } = string.Empty;
        public string Lastname { get; set; } = string.Empty;
        public List<int>? ProfileImageBytes { get; set; }
    }
}
